src/content/docs/en/guides/cms/flotiq.mdx

```diff
@@ -0,0 +1,280 @@
+import { flotiq } from "../../lib/flotiq";
+
+export async function getStaticPaths() {
+  const posts = await flotiq.BlogpostAPI.list();
+  return posts.data?.map((post) => ({
+    params: { slug: post.slug },
+    props: post
+  })) || []
+}
+const post: Blogpost = Astro.props;
+---
+<html lang="en">
+  <title>{post.title}</title>
+  <body>
+    <h1>{post.title}</h1>
+    <div set:html={post.content}/>
+  </body>
+</html>
+```
+
+Now when you visit `http://localhost:4321` and click on a blog post, you will be able to navigate to the individual post's page.
+
+#### On-demand Rendering
+
+If you are using [SSR](/en/guides/server-side-rendering/) mode, you will need to fetch a single post based on its `slug`.
+
+Create a new file `[slug].astro` in the `/src/pages/posts/` directory. Fetch the post by its `slug` field, including logic to display a 404 page when the route is not found:
+
+```astro title="src/pages/posts/[slug].astro"
+---
+import type { Blogpost } from "flotiq-api-ts";
+import { flotiq } from "../../lib/flotiq";
+
+const { slug } = Astro.params;
+let post: Blogpost;
+
+const blogpostList = await flotiq.BlogpostAPI.list({
+  filters: JSON.stringify({
+    slug: {
+      type: 'equals',
+      filter: slug,
+    }
+  }),
+  limit: 1
+});
+
+if (blogpostList.data?.[0]) {
+  post = blogpostList.data[0]
+} else {
+  return Astro.redirect('/404');
+}
+---
+```
+
+Next, add the templating to display an individual post:
+
+```astro title="src/pages/posts/[slug].astro" ins={24-30}
+---
+import type { Blogpost } from "flotiq-api-ts";
+import { flotiq } from "../../lib/flotiq";
+
+const { slug } = Astro.params;
+let post: Blogpost;
+
+const blogpostList = await flotiq.BlogpostAPI.list({
+  filters: JSON.stringify({
+    slug: {
+      type: 'equals',
+      filter: slug,
+    }
+  }),
+  limit: 1
+});
+
+if (blogpostList.data?.[0]) {
+  post = blogpostList.data[0]
+} else {
+  return Astro.redirect('/404');
+}
+---
+<html lang="en">
+  <title>{post.title}</title>
+  <body>
+    <h1>{post.title}</h1>
+    <div set:html={post.content}/>
+  </body>
+</html>
+```
+
+Now when you visit `http://localhost:4321` and click on a blog post, you will be able to navigate to the individual post's page.
+
+### Refreshing the SDK After Content Type Changes
+
+When using the Flotiq TypeScript SDK (`flotiq-api-ts`), all your data types are accurately mapped into the Astro project.
+
+If you make changes to the structure of your content types (such as adding a new field or modifying an existing one), you’ll need to refresh the SDK to ensure that your project reflects the latest model updates.
+
+To do this, run the rebuild command for your package manager:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+    ```sh
+    npm rebuild flotiq-api-ts
+    ```
+  </Fragment>
+  <Fragment slot="pnpm">
+    ```sh
+    pnpm rebuild flotiq-api-ts
+    ```
+  </Fragment>
+  <Fragment slot="yarn">
+    ```sh
+    yarn add flotiq-api-ts
```

Comment:
shouldn't this be rebuild? or am i missing something

```suggestion
    yarn rebuild flotiq-api-ts
```

Reply:
Thanks for the suggestion! The proposed `yarn add` works and rebuilds the package each time (which is what we expect), but this applies specifically to yarn v1. In yarn v4, the appropriate command for this is `yarn rebuild`.

I suggest updating the line to `yarn rebuild flotiq-api-ts` (as you suggested) and optionally adding a note:
"If you are using yarn v1 (Classic), run `yarn add flotiq-api-ts` to rebuild the SDK."

Reply:
Yes, it can be done! Perhaps let's get another opinion on this, but what you proposed works for me. It will help avoid confusion for the end user who sees one method for all the other package managers and a different one for Yarn.